staging/src/k8s.io/kubectl/pkg/cmd/cp/cp.go

```diff
@@ -356,13 +368,13 @@ func makeTar(srcPath, destPath string, writer io.Writer) error {
 	tarWriter := tar.NewWriter(writer)
 	defer tarWriter.Close()
 
-	srcPath = path.Clean(srcPath)
-	destPath = path.Clean(destPath)
-	return recursiveTar(path.Dir(srcPath), path.Base(srcPath), path.Dir(destPath), path.Base(destPath), tarWriter)
+	srcPath = filepath.Clean(srcPath)
+	destPath = filepath.Clean(destPath)
+	return recursiveTar(filepath.Dir(srcPath), filepath.Base(srcPath), filepath.Dir(destPath), filepath.Base(destPath), tarWriter)
 }
 
 func recursiveTar(srcBase, srcFile, destBase, destFile string, tw *tar.Writer) error {
-	srcPath := path.Join(srcBase, srcFile)
+	srcPath := filepath.Join(srcBase, srcFile)
 	matchedPaths, err := filepath.Glob(srcPath)
 	if err != nil {
 		return err
```

Comment:
does this change mean `matchedPaths` could be `\`-separated on Windows? do we need to ensure the path passed to `tar.FileInfoHeader(stat, fpath)` is a unix path?